packages/opentelemetry/src/trace.ts

```diff
@@ -45,8 +45,6 @@ export function startSpan<T>(options: OpenTelemetrySpanContext, callback: (span:
     const spanOptions = getSpanOptions(options);
 
     return tracer.startActiveSpan(name, spanOptions, ctx, span => {
-      _applySentryAttributesToSpan(span, options);
```

Comment:
I noticed that this was actually also not ideal, we just added the op attribute after the span was started, which means this was not sent to the sampler. Also, this is overall unnecessary, we can just add the attribute right away!